homeassistant/components/device_tracker/locative.py

```diff
@@ -76,21 +77,24 @@ def _handle(self, data):
         device = data['device'].replace('-', '')
         location_name = data['id'].lower()
         direction = data['trigger']
+        
+        kwargs = {
+           'dev_id': device,
+           'location_name': location_name,
+           'gps': (data[ATTR_LATITUDE], data[ATTR_LONGITUDE])
+        }
 
         if direction == 'enter':
-            yield from self.hass.loop.run_in_executor(
-                None, partial(self.see, dev_id=device,
-                              location_name=location_name))
+            self.see(**kwargs)
```

Comment:
I think you still have to use "yield from"
https://travis-ci.org/home-assistant/home-assistant/jobs/172768411#L1704